docsrc/getting_started/getting_started_with_windows.rst

```diff
@@ -7,133 +7,28 @@ Torch-TensorRT has community support for Windows platform using CMake
 
 Prerequisite:
 
-* Microsoft Visual Studio
-* LibTorch
-* TensorRT
+* Microsoft VS 2022 Tools
+* Bazelisk
 * CUDA
 
 
-Build configuration
+Build steps
 -------------------
 
-* Open Microsoft Visual Studio
-* Open Torch-TensorRT source code folder
-* Open Manage configurations -> Edit JSON to open CMakeSettings.json file.
-* Configure the CMake build configurations. Following is an example configuration:
-
-.. code-block:: none
```

Comment:
Is it useful to leave the old Visual Studio instructions in this tutorial?

Reply:
The existing windows support was using cmake right ?